jvm/testutils/deps.bzl

```diff
@@ -11,11 +11,11 @@ main_exports = [
     "//jvm/j2v8:j2v8-all",
     "//jvm/graaljs",
     "//plugins/common-types/jvm:common-types",
-    "//plugins/reference-assets/jvm:reference-assets",
+    "//plugins/mocks:jar",
 ]
 
 main_deps = main_exports
 
 main_resources = [
-    "//plugins/reference-assets/mocks",
+    "//plugins/mocks:jar",
```

Comment:
I don't think this is necessary anymore if you're defining the JAR as a dep. It's more for when you have actual files to include within a JAR.